src/inject.js

```diff
@@ -219,19 +219,29 @@ export const injectAndGetClassName = (
 ) /* : string */ => {
     styleDefinitions = flattenDeep(styleDefinitions);
 
-    // Filter out falsy values from the input, to allow for
-    // `css(a, test && c)`
-    const validDefinitions = styleDefinitions.filter((def) => def);
-
+    let classNameBits = [];
+    let definitionBits = [];
+    for (let i = 0; i < styleDefinitions.length; i += 1) {
+      // Filter out falsy values from the input, to allow for
+      // `css(a, test && c)`
+      if (styleDefinitions[i]) {
+        classNameBits.push(styleDefinitions[i]._name);
+        definitionBits.push(styleDefinitions[i]._definition);
+      }
+    }
```

Comment:
gah! Would you be opposed to me setting up eslint to tell me about these things for this project?